code/game/gamemodes/shadowling/special_shadowling_abilities.dm

```diff
@@ -33,6 +33,8 @@ var/list/possibleShadowlingNames = list("U'ruan", "Y`shej", "Nex", "Hel-uae", "N
 			for(var/obj/structure/alien/resin/wall/shadowling/R in shadowturf) //extremely hacky
 				qdel(R)
 				new /obj/structure/alien/weeds/node(shadowturf) //Dim lighting in the chrysalis -- removes itself afterwards
+			var/temp_flags = H.status_flags
+			H.status_flags |= GODMODE //Can't die while hatching
```

Comment:
@YotaXP usually it's enough for him to just point things out that other maintainers missed, cuz we can all tell exactly what's bad about it and let the person know